src/utils/typeguards.ts

```diff
@@ -156,3 +186,165 @@ export function isVenue(value: unknown): value is Venue {
+  return [
+    ClaimType.NoData,
+    ClaimType.CustomerDueDiligence,
+    ClaimType.InvestorUniquenessV2,
+  ].includes(type);
+}
+
+/**
+ * Return whether Claim is an AccreditedClaim
+ */
+export function isAccreditedClaim(claim: Claim): claim is AccreditedClaim {
+  return claim.type === ClaimType.Accredited;
+}
+
+/**
+ * Return whether Claim is an AffiliateClaim
+ */
+export function isAffiliateClaim(claim: Claim): claim is AffiliateClaim {
+  return claim.type === ClaimType.Affiliate;
+}
+
+/**
+ * Return whether Claim is an BuyLockupClaim
+ */
+export function isBuyLockupClaim(claim: Claim): claim is BuyLockupClaim {
+  return claim.type === ClaimType.BuyLockup;
+}
+
+/**
+ * Return whether Claim is a SellLockupClaim
+ */
+export function isSellLockupClaim(claim: Claim): claim is SellLockupClaim {
+  return claim.type === ClaimType.SellLockup;
+}
+
+/**
+ * Return whether Claim is a CustomerDueDiligenceClaim
+ */
+export function isCustomerDueDiligenceClaim(claim: Claim): claim is CddClaim {
+  return claim.type === ClaimType.CustomerDueDiligence;
+}
+
+/**
+ * Return whether Claim is a KnowYourCustomerClaim
+ */
+export function isKnowYourCustomerClaim(claim: Claim): claim is KnowYourCustomerClaim {
+  return claim.type === ClaimType.KnowYourCustomer;
+}
+
+/**
+ * Return whether Claim is a JurisdictionClaim
+ */
+export function isJurisdictionClaim(claim: Claim): claim is JurisdictionClaim {
+  return claim.type === ClaimType.Jurisdiction;
+}
+
+/**
+ * Return whether Claim is a ExemptClaim
+ */
+export function isExemptClaim(claim: Claim): claim is ExemptClaim {
+  return claim.type === ClaimType.Exempted;
+}
+
+/**
+ * Return whether Claim is a BlockedClaim
+ */
+export function isBlockedClaim(claim: Claim): claim is BlockedClaim {
+  return claim.type === ClaimType.Blocked;
+}
+
+/**
+ * Return whether a Claim is an InvestorUniquenessClaim
+ */
+export function isInvestorUniquenessClaim(claim: Claim): claim is InvestorUniquenessClaim {
+  return claim.type === ClaimType.InvestorUniqueness;
+}
+
+/**
+ * Return whether Claim is a NoDataClaim
+ */
+export function isNoDataClaim(claim: Claim): claim is NoDataClaim {
+  return claim.type === ClaimType.NoData;
+}
+
+/**
+ *  Return whether a Claim is an InvestorUniquenessV2Claim
+ */
+export function isInvestorUniquenessV2Claim(claim: Claim): claim is InvestorUniquenessV2Claim {
+  return claim.type === ClaimType.InvestorUniquenessV2;
+}
+
+/**
+ * Return whether Condition has a single Claim
+ */
+export function isSingleClaimCondition(
+  condition: InputCondition
+): condition is InputConditionBase & SingleClaimCondition {
+  return [ConditionType.IsPresent, ConditionType.IsAbsent].includes(condition.type);
+}
+
+/**
+ * Return whether Condition has multiple Claims
+ */
+export function isMultiClaimCondition(
+  condition: InputCondition
+): condition is InputConditionBase & MultiClaimCondition {
+  return [ConditionType.IsAnyOf, ConditionType.IsNoneOf].includes(condition.type);
+}
```

Comment:
These two should probably check for the existence of the `claim` and `claims` properties instead of the type. Shouldn't matter much, but it's a bit more future proof in case we start adding more condition types